packages/database/src/migrations/20210606024543-CreateDashboardItemsTableLayout-modifies-schema.js

```diff
@@ -20,8 +20,7 @@ exports.up = async function (db) {
       id TEXT PRIMARY KEY,
       code TEXT NOT NULL UNIQUE,
       config JSONB NOT NULL DEFAULT '{}',
-      permission_groups TEXT[] NOT NULL CONSTRAINT permission_groups_not_empty CHECK (permission_groups <> '{}'),
-      report_code TEXT NOT NULL,
+      report_code TEXT,
```

Comment:
I've removed NOT NULL constraint here because there are a few static dashboard_item that don't fetch data at all (eg: no_data_at_level dashboardReport)